include/usb/class/usb_hid.h

```diff
@@ -157,6 +160,9 @@ void usb_hid_register_device(const u8_t *desc, size_t size,
 /* Write to hid interrupt endpoint */
 int hid_int_ep_write(const u8_t *data, u32_t data_len, u32_t *bytes_ret);
 
+/* Read from hid interrupt endpoint */
+int hid_int_ep_read(u8_t *data, u32_t max_data_len, u32_t *ret_bytes);
```

Comment:
Minor: Please make the variable names consistent between read and write.

Reply:
@sathishkuttan, the variable names are consistent with `usb_write` and `usb_read`. I think that's what matters here.

Reply:
Neither one matters, the consistency is just cosmetic and is a good coding practice.